beetsplug/missing.py

```diff
@@ -106,35 +111,95 @@ def __init__(self):
         self._command.parser.add_option(
             u'-t', u'--total', dest='total', action='store_true',
             help=u'count total of missing tracks')
+        self._command.parser.add_option(
+            u'-a', u'--albums', dest='albums', action='store_true',
```

Comment:
In other commands, such as `beet ls`, a similar (but different) flag is always called `--album`, not `--albums`. Can we please stick with that name for consistency? (And the same for the config option name, of course.)